### ISSUE TITLE
Fixed Length Records should read records based on record length

### ISSUE THREAD

**Issue by shariqueshueb** [MarcosMeli/FileHelpers#188]
The purpose of fixed length is defeated when you start using the new line character as a record separator. Instead of the 'NewLineDelimitedRecordReader' a FixedLengthRecordReader should be used.

**Comment by breusshe** [comment]
I agree with @shariqueshueb.  Fixed width records will have an "end record" delimiter.  For instance, I'm using one now that uses the pipe character ( | ).  So, we should have a way of setting the end record delimiter and having that mark where to transition to a new record rather than assume the new line character is it.

**Comment by alanschrank** [comment]
To add to this, fixed length records don't have to have any record delimiter. Character 1 of record n could start right after the last character of record n-1
